store/works.js

```diff
@@ -10,17 +10,17 @@ class Works extends Pages {
   }
 
   @action
-  async changeVisibility(rowId, isVisible) {
+  async changeVisibility(rowId) {
     const [pageNumber, workId] = rowId.split('-', 2)
     // TODO: consider moving this endpoint to /data-providers/works/
-    await apiRequest(`/works/${workId}`, {
-      method: 'PATCH',
-      body: { disabled: !isVisible },
-    })
+    // await apiRequest(`/works/${workId}`, {
+    //   method: 'PATCH',
+    //   body: { disabled: !isVisible },
+    // })
```

Comment:
How did this comment appear?

Reply:
Once I finish I am gonna uncomment it. I am just afraid to call that endpoint while testing because I may lose track of which article I disabled. Hopefully, if we have a separate database for staging I won't need to do this while developing.